app/inbound_number/rest.py

```diff
@@ -0,0 +1,64 @@
+from flask import Blueprint, jsonify, request
+
+from app.dao.inbound_numbers_dao import (
+    dao_get_inbound_numbers,
+    dao_get_inbound_number,
+    dao_get_inbound_number_for_service,
+    dao_get_available_inbound_numbers,
+    dao_set_inbound_number_to_service,
+    dao_set_inbound_number_active_flag
+)
+from app.errors import InvalidRequest, register_errors
+from app.models import InboundNumber
+from app.schema_validation import validate
+
+inbound_number_blueprint = Blueprint('inbound_number', __name__)
```

Comment:
Not something I want you to change now, but in future urls with two words should be separated with a `-` rather than  an `_`